java-rest-service/accelerator.yaml

```diff
@@ -57,22 +52,22 @@ accelerator:
     - name: apiSystem
       label: System API Belongs To
       inputType: text
-      required: false
-      defaultValue: ''
+      required: true
+      defaultValue: "Profile Management"
```

Comment:
system names that I have seen are lowercase and kebob-case - see https://backstage.io/docs/features/software-catalog/descriptor-format#kind-system

Reply:
Done